galley/pkg/source/kube/dynamic/converter/converter_test.go

```diff
@@ -572,113 +570,3 @@ func TestShouldProcessIngress(t *testing.T) {
 		}
 	}
 }
-
```

Comment:
Why is this deleted?

Reply:
Correction, this was deleted because my code replaces the previous work on generating ServiceEntry.  The conversion logic has been moved out of the source and now appears further along in the pipeline.